### ISSUE TITLE
Contrast issues on homepage

### ISSUE THREAD

**Issue by mattfowles1994** [mattfowles1994/ArtGallery#7]
The contrast rating on the navbar links does not meet WCAG AA ratings. Preferably this would reach AAA rating.
![image](https://user-images.githubusercontent.com/22026984/115611915-9c171880-a2e2-11eb-9a24-87f1eed0da5c.png)

Steps to reproduce:
- This test was reached using the WAVE extension

**Comment by mattfowles1994** [comment]
The CSS has been edited to slightly lighten the text and increase the contrast. Please see below new scores;
![image](https://user-images.githubusercontent.com/22026984/115613422-47749d00-a2e4-11eb-86b6-ad3695ee7f64.png)
